pkg/klusterlet/clusterinfo/clusterinfo_controller.go

```diff
@@ -388,6 +398,72 @@ func (r *ClusterInfoReconciler) getDistributionInfo() (clusterv1beta1.Distributi
 	return distributionInfo, nil
 }
 
+func distributionInfoUpdated(old, new *clusterv1beta1.DistributionInfo) bool {
+	switch new.Type {
+	case clusterv1beta1.DistributionTypeOCP:
+		return ocpDistributionInfoUpdated(&old.OCP, &new.OCP)
+	}
+	return false
+}
+
+func ocpDistributionInfoUpdated(old, new *clusterv1beta1.OCPDistributionInfo) bool {
+	if len(old.AvailableUpdates) != len(new.AvailableUpdates) {
+		return true
+	}
+
+	var sortedAvailableUpdates []string
```

Comment:
I think you could use  similar of this https://github.com/open-cluster-management/work/blob/main/pkg/spoke/controllers/appliedmanifestcontroller/appliedmanifestwork_controller.go#L167-L180 to sort?

Reply:
done